sdk/core/core-util/review/core-util.api.md

```diff
@@ -23,13 +23,14 @@ export interface CreateAbortablePromiseOptions {
 }
 
 // @public
-export function delay(timeInMs: number, options?: DelayOptions): Promise<void>;
+export function delay(timeInMs: number, options?: DelayOptions_2): Promise<void>;
 
 // @public
-export interface DelayOptions {
+interface DelayOptions_2 {
     abortErrorMsg?: string;
     abortSignal?: AbortSignalLike;
 }
+export { DelayOptions_2 as DelayOptions }
```

Comment:
this is strange, why did api-extractor do this to us?

Reply:
Chatted with Matt offline, it is because of [dom being imported](https://github.com/microsoft/TypeScript/blob/ae2b4afdcbb80d291f94f78b81a1588c041459ab/src/lib/dom.generated.d.ts#L348).

Reply:
Fixed